shuffle/cli/src/test.rs

```diff
@@ -130,3 +160,73 @@ fn run_deno_test(project_path: &Path, config: &NodeConfig) -> Result<()> {
         .wait()?;
     Ok(())
 }
+
+fn run_move_unit_tests(project_path: &Path) -> Result<()> {
+    let unit_test_cmd = cli::PackageCommand::UnitTest {
+        // Setting to default values with exception of report_storage_on_error
+        // to get more information about a failed test
+        instruction_execution_bound: 5000,
+        filter: None,
+        list: false,
+        num_threads: 8,
```

Comment:
Is there a default setting that's more aware of the available CPUs/cores? Perhaps the default handles this?

Often times when deciding on the # of threads, one will check the underlying machine for the # of available CPUs. 8 for example could be overkill for a single core machine. If this is readily available in the repo, let's use it. Otherwise, let's leave it as is.

Reply:
The move team set the default value to 8. https://github.com/diem/diem/blob/main/language/tools/move-cli/src/package/cli.rs#L81

Maybe @tzakian could answer this question.

Reply:
You can use the [num_cpus](https://crates.io/crates/num_cpus) crate to get this if you want, but it might be a bit of overkill. I decided on `8` as a default as a decent middle-ground, but the number was chosen somewhat arbitrarily, so feel free to change it to something else if you think it will make more sense :)